### ISSUE TITLE
Consider switching to MapBox or Google Maps

### ISSUE THREAD

**Issue by iandundas** [iandundas/Tacks#47]
https://www.mapbox.com/ios-sdk/#map_view__class

**Comment by iandundas** [comment]
Consider adding a cool map skin, also see what else they can do

**Comment by iandundas** [comment]
Downside of Mapbox when using free plan - it requires background location tracking, which seems a bit sketchy to me. Is that how free is paid for, by tracking the users?

**Comment by jordibruin** [comment]
That IS a little sketchy. Is it only required for the free plan?

**Comment by iandundas** [comment]
Yep - all other plans can have app-active location tracking only 

> Enabling background location data
> All free (Starter plan) mobile developers are required to enable background location data

https://www.mapbox.com/guides/first-steps-ios-sdk/#background-location

:(

**Comment by iandundas** [comment]
> We just added three new guides to help you design beautiful maps in Mapbox Studio.
> Start with Getting started with Mapbox Studio http://email.mapbox.com/e/c/eyJlbWFpbF9pZCI6Ik5Ea3pPVG9WU1FKa0FBSnpBQllKUHhvQlJNM1BJbWNWU0JoV1lFcmdBVzA2TWpVMU9UQXhBQT09IiwicG9zaXRpb24iOjEsImhyZWYiOiJodHRwczovL3d3dy5tYXBib3guY29tL2hlbHAvZ2V0dGluZy1zdGFydGVkLW1hcGJveC1zdHVkaW8tMS8_dXRtX3NvdXJjZT1ndWlkZXNfc3RhcnRlciZ1dG1fbWVkaXVtPWVtYWlsJnV0bV9jb250ZW50PXN0dWRpb19ndWlkZSZ1dG1fY2FtcGFpZ249Zmlyc3RndWlkZSIsImxpbmtfaWQiOjk3ODY4MzR9/89b96b588ca12001749d44147f30926a9e4c16220d5897d167d43bc29670291f for a complete walk-through. Get acquainted with the tools for inspecting styles, uploading data, and managing your account.
> 
> Next, learn the basics of designing a map and find productivity tips to help you work effectively in Getting started with the Mapbox Studio style editor http://email.mapbox.com/e/c/eyJlbWFpbF9pZCI6Ik5Ea3pPVG9WU1FKa0FBSnpBQllKUHhvQlJNM1BJbWNWU0JoV1lFcmdBVzA2TWpVMU9UQXhBQT09IiwicG9zaXRpb24iOjIsImhyZWYiOiJodHRwczovL3d3dy5tYXBib3guY29tL2hlbHAvZ2V0dGluZy1zdGFydGVkLW1hcGJveC1zdHVkaW8tMi8_dXRtX3NvdXJjZT1lZGl0b3JfZ3VpZGUmdXRtX21lZGl1bT1lbWFpbCZ1dG1fY29udGVudD1zdHVkaW9fZ3VpZGUmdXRtX2NhbXBhaWduPXNlY29uZGd1aWRlIiwibGlua19pZCI6OTc4NjgzNX0/2766942092e03e53275523995677e77697b30d92e8933ca8c197be7d1c4ec89b.
> 
> Finally, learn all about publishing your Mapbox Studio map style http://email.mapbox.com/e/c/eyJlbWFpbF9pZCI6Ik5Ea3pPVG9WU1FKa0FBSnpBQllKUHhvQlJNM1BJbWNWU0JoV1lFcmdBVzA2TWpVMU9UQXhBQT09IiwicG9zaXRpb24iOjMsImhyZWYiOiJodHRwczovL3d3dy5tYXBib3guY29tL2hlbHAvZ2V0dGluZy1zdGFydGVkLW1hcGJveC1zdHVkaW8tMi8_dXRtX3NvdXJjZT1lZGl0b3JfZ3VpZGUmdXRtX21lZGl1bT1lbWFpbCZ1dG1fY29udGVudD1zdHVkaW9fZ3VpZGUmdXRtX2NhbXBhaWduPXNlY29uZGd1aWRlIiwibGlua19pZCI6OTc4NjgzNX0/1d9f746a2fee701c50fa0c21925296b6e443bf05239a7f65fb9b263932d01444 to add your map to mobile apps or websites.
> We can't wait to see what you build!
> Love,
> Your friends at Mapbox
